Gems/PhysX/Code/Source/EditorHeightfieldColliderComponent.h

```diff
@@ -75,6 +76,7 @@ namespace PhysX
         AzPhysics::SimulatedBody* GetSimulatedBody() override;
         AzPhysics::SimulatedBodyHandle GetSimulatedBodyHandle() const override;
         AzPhysics::SceneQueryHit RayCast(const AzPhysics::RayCastRequest& request) override;
+        AzPhysics::SimulatedBody* GetSimulatedBody() const;
```

Comment:
The const function of GetSimulatedBody is missing to return a const pointer of SimulatedBody, otherwise the signature of the two GetSimulatedBody functions are the same.

sounds like the const version of GetSimulatedBody could be added to the bus `SimulatedBodyComponentRequestsBus`. Otherwise seems odd putting it beside the bus overrides and not be part of it.

Reply:
It doesn't seem to be possible to have 2 functions with the same name in the EBus.. 
Otherwise yes, it'd make sense.

Reply:
Got it, thanks.

Reply:
(the ugly solution to this which is sometimes necessary is to have `GetThingConst` (to avoid needing to overload them) - not ideal but sometimes it's better to be explicit and technically correct

Reply:
It is possible to have functions with the same name as part of the EBus interface class.
Calling the function using the member function pointer syntax requires a cast.
Using a lambda however does not.